src/core/reducers/user.js

```diff
@@ -30,6 +32,17 @@ export const isAuthenticated = (state: { user: UserStateType }) => {
   return !!state.user.id;
 };
 
+export const selectDisplayName = (state: { user: UserStateType }) => {
+  const displayName = state.user.displayName;
+  if (displayName && displayName !== '') {
```

Comment:
I always like doing a `displayName.length` check for empty strings instead, but that's just me. I tend to use that idiom in the code but if it's varied do whichever you like 😄